.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "stable"
-  - "5"
-  - "4"
+  - 6
+  - 4
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm i -g npm@latest
```

Comment:
Aha, that's ok. Could you update your PR so we could merge it? And what about other questions?